examples/wave_example.ipynb

```diff
@@ -25,154 +25,1751 @@
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 47 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                     0.0200  0.0325  0.0375  0.0425  0.0475  0.0525  0.0575  \\\n",
+       "2018-01-01 00:40:00     0.0     0.0     0.0     0.0     0.0    0.03    0.04   \n",
+       "2018-01-01 01:40:00     0.0     0.0     0.0     0.0     0.0    0.02    0.06   \n",
+       "2018-01-01 02:40:00     0.0     0.0     0.0     0.0     0.0    0.00    0.08   \n",
+       "2018-01-01 03:40:00     0.0     0.0     0.0     0.0     0.0    0.00    0.13   \n",
+       "2018-01-01 04:40:00     0.0     0.0     0.0     0.0     0.0    0.03    0.14   \n",
+       "\n",
+       "                     0.0625  0.0675  0.0725  ...  0.3300  0.3400  0.3500  \\\n",
+       "2018-01-01 00:40:00    0.09    0.22    0.22  ...    0.07    0.08    0.07   \n",
+       "2018-01-01 01:40:00    0.08    0.21    0.21  ...    0.07    0.07    0.08   \n",
+       "2018-01-01 02:40:00    0.07    0.14    0.32  ...    0.08    0.06    0.05   \n",
+       "2018-01-01 03:40:00    0.22    0.26    0.32  ...    0.05    0.06    0.05   \n",
+       "2018-01-01 04:40:00    0.17    0.37    0.40  ...    0.06    0.12    0.04   \n",
+       "\n",
+       "                     0.3650  0.3850  0.4050  0.4250  0.4450  0.4650  0.4850  \n",
+       "2018-01-01 00:40:00    0.07    0.03    0.01    0.02    0.01    0.01     0.0  \n",
+       "2018-01-01 01:40:00    0.02    0.01    0.02    0.01    0.01    0.00     0.0  \n",
+       "2018-01-01 02:40:00    0.03    0.03    0.02    0.01    0.01    0.00     0.0  \n",
+       "2018-01-01 03:40:00    0.03    0.03    0.01    0.01    0.00    0.00     0.0  \n",
+       "2018-01-01 04:40:00    0.04    0.02    0.01    0.01    0.01    0.01     0.0  \n",
+       "\n",
+       "[5 rows x 47 columns]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Set the random seed, to reproduce results\n",
-    "np.random.seed(1) \n",
+    "ndbc_data_file = '../mhkit/tests/data/data.txt'\n",
+    "\n",
+    "[raw_ndbc_data, meta] = wave.io.read_NDBC_file(ndbc_data_file) # read_NDBC_file outputs the NDBC file data into two variables.\n",
+    "                                            # data is a pandas DataFrame containing the file data. \n",
+    "                                            # meta contains the meta data, if available. \n",
     "\n",
-    "Hm0 = pd.Series(np.random.rayleigh(2, 1000000)) # significant wave height values\n",
-    "Te = pd.Series(np.random.normal(4.5, .7, 1000000)) # peak period values\n",
-    "P = pd.Series(np.random.normal(200, 40, 1000000)) # power values\n",
-    "J = pd.Series(np.random.normal(300, 8, 1000000)) # omnidirectional wave flux values"
+    "raw_ndbc_data.head()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Capture Length Matrices\n",
```

Comment:
This is where the Capture Length Matrices section was removed.

Reply:
@kmruehl  Is Capture length Matrices on "In [8]" of https://github.com/rpauly18/MHKiT-Python-2/blob/cherry-branch/examples/wave_example.ipynb ???

Reply:
@ssolson yes, it's there. I'm not sure why it looked like it had been removed in the commit history. I think my confusion may be related to the way Github renders ipynb files in the PR. The section is still on @rpauly18's fork, so it should be fine.